src/api/app/controllers/source_controller.rb

```diff
@@ -302,9 +302,9 @@ class NoMatchingReleaseTarget < APIException
   def verify_can_modify_target_package!
     return if User.current.can_modify_package?(@package)
 
-    raise CmdExecutionNoPermission, "no permission to execute command '#{params[:cmd]}' " +
+    raise CmdExecutionNoPermission, "no permission to execute command '#{params[:cmd]}' " \
```

Comment:
I can live with this change, but I don't really understand it... I need to look up the reasoning for why a backslash is preferred here.